audio/libsoundio/Portfile

```diff
@@ -54,11 +57,10 @@ variant jack description {Enable Jack support} {
     configure.args-replace  -DENABLE_JACK=OFF -DENABLE_JACK=ON
 }
 
-# error: use of undeclared identifier 'kAudioObjectPropertyScopeInput'
-# 10.8+ https://developer.apple.com/documentation/coreaudio/1494464-anonymous/kaudioobjectpropertyscopeinput?language=objc
-# https://github.com/andrewrk/libsoundio/issues/290
+# There is no need to prohibit CoreAudio now,
+# but mileage may vary. While PulseAudio also has issues,
+# those are likely to be sorted out.
 if {${os.platform} eq "darwin" && ${os.major} < 12} {
-    configure.args-replace  -DENABLE_COREAUDIO=ON  -DENABLE_COREAUDIO=OFF
     default_variants-append +pulseaudio
 }
```

Comment:
This block of code was added in ab5c8334abea7a6e77794e301a91f1ae3b9bc607 to use pulseaudio on old systems where coreaudio could not be used. Now that you've fixed coreaudio for old systems, there's no remaining need to enable pulseaudio on those systems, is there?

Reply:
Yeah, let me remove it. Especially given pulseaudio itself has to be fixed now.

Reply:
@ryandesign This one is sorted.